sql/core.go

```diff
@@ -625,7 +625,8 @@ type Closer interface {
 
 // RowReplacer is a combination of RowDeleter and RowInserter.
 // TODO: We can't embed those interfaces because go 1.13 doesn't allow for overlapping interfaces (they both declare
-//  Close). Go 1.14 fixes this problem, but we aren't ready to drop support for 1.13 yet.
+//
+//	Close). Go 1.14 fixes this problem, but we aren't ready to drop support for 1.13 yet.
```

Comment:
some of these comments look bad now. need to find a way around the CI formatter, if that's what's causing these